### PR TITLE
add registry.redhat login instructions

### PR DESCRIPTION
**Pull request by leena-jawale**
issue: https://issues.redhat.com/browse/AAP-13098

### CODE REVIEW

File: stories/topics/proc-aws-backup-platform-stack.adoc [hunk 1]
```diff
@@ -6,6 +6,16 @@
 . Pull the `ansible-on-clouds-ops 2.2` container image.
 +
 [NOTE]
+====
+Before Pulling the docker image, make sure you are login to redhat registry using docker. Use the below command to login to redhat registry.
```
Comment (chrisahl) [inline stories/topics/proc-aws-backup-platform-stack.adoc:10]: Will need to fix all the additions that have "make sure you are login to redhat registry"
to either
"make sure you login to redhat registry"
OR
"make sure you are logged in to redhat registry"
@ianf77 @ariordan-redhat can help with what text they want to use

File: stories/topics/proc-aws-set-container-image.adoc [hunk 2]
```diff
@@ -10,7 +10,7 @@ For example, if your foundation deployment version is 2.3.20230221-00, pull the
 +
 [NOTE]
 ====
-Before Pulling the docker image, make sure you are login to redhat registry using docker. Use the below command to login to redhat registry. 
+Before Pulling the docker image, make sure you are logged in to redhat registry using docker. Use the below command to login to redhat registry.
```
Comment (ianf77) [inline stories/topics/proc-aws-set-container-image.adoc:13]: Use the following command to login to red hat registry (registry.redhat.com?)

File: stories/topics/proc-gcp-from-stack-pull-container-image.adoc [hunk 3]
```diff
@@ -7,7 +7,7 @@
 +
 [NOTE]
 ====
-Before Pulling the docker image, make sure you are login to redhat registry using docker. Use the below command to login to redhat registry. 
+Before Pulling the docker image, make sure you are logged in to redhat registry using docker. Use the below command to login to redhat registry.
```
Comment (ianf77) [inline stories/topics/proc-gcp-from-stack-pull-container-image.adoc:10]: As above

File: stories/topics/proc-gcp-pull-backup-container-image.adoc [hunk 4]
```diff
@@ -7,7 +7,7 @@
 +
 [NOTE]
 ====
-Before Pulling the docker image, make sure you are login to redhat registry using docker. Use the below command to login to redhat registry. 
+Before Pulling the docker image, make sure you are logged in to redhat registry using docker. Use the below command to login to redhat registry.
```
Comment (ianf77) [inline stories/topics/proc-gcp-pull-backup-container-image.adoc:10]: As above

File: stories/topics/proc-gcp-upgrade-pull-container-image.adoc [hunk 5]
```diff
@@ -7,7 +7,7 @@
 +
 [NOTE]
 ====
-Before Pulling the docker image, make sure you are login to redhat registry using docker. Use the below command to login to redhat registry. 
+Before Pulling the docker image, make sure you are logged in to redhat registry using docker. Use the below command to login to redhat registry.
```
Comment (ianf77) [inline stories/topics/proc-gcp-upgrade-pull-container-image.adoc:10]: As above

**Review by chrisahl — APPROVED** [review]

File: stories/topics/proc-aws-backup-platform-stack.adoc [hunk 6]
```diff
@@ -6,6 +6,16 @@
 . Pull the `ansible-on-clouds-ops 2.2` container image.
 +
 [NOTE]
+====
+Before Pulling the docker image, make sure you are logged in to redhat registry using docker. Use the following command to login to registry.redhat.com. 
+[source,bash]
+----
+$ docker login registry.redhat.io
+----
+Refer link:https://access.redhat.com/RegistryAuthentication[Registry Authentication] for more information about registry login.
```
Comment (ianf77) [inline stories/topics/proc-aws-backup-platform-stack.adoc:15]: For more information about registry login, see link....

File: stories/topics/proc-aws-from-stack-pull-container-image.adoc [hunk 7]
```diff
@@ -5,6 +5,16 @@
...
 . Pull the `ansible-on-clouds-ops` 2.2 container image with the same tag as the foundation deployment.
 +
+[NOTE]
+====
+Before Pulling the docker image, make sure you are logged in to redhat registry using docker. Use the following command to login to registry.redhat.com.
+[source,bash]
+----
+$ docker login registry.redhat.io
+----
+Refer link:https://access.redhat.com/RegistryAuthentication[Registry Authentication] for more information about registry login.
```
Comment (ianf77) [inline stories/topics/proc-aws-from-stack-pull-container-image.adoc:15]: As above.

File: stories/topics/proc-aws-set-container-image.adoc [hunk 8]
```diff
@@ -8,6 +8,16 @@ For example, if your foundation deployment version is 2.3.20230221-00, pull the
...
 . Pull the `ansible-on-clouds-ops` container image with the same tag version as the foundation deployment.
 +
+[NOTE]
+====
+Before Pulling the docker image, make sure you are logged in to redhat registry using docker. Use the following command to login to registry.redhat.com. 
+[source,bash]
+----
+$ docker login registry.redhat.io
+----
+Refer link:https://access.redhat.com/RegistryAuthentication[Registry Authentication] for more information about registry login.
```
Comment (ianf77) [inline stories/topics/proc-aws-set-container-image.adoc:18]: As above.

File: stories/topics/proc-aws-upgrade-pull-container-image.adoc [hunk 9]
```diff
@@ -7,6 +7,16 @@
 +
 [NOTE]
 ====
+Before Pulling the docker image, make sure you are logged in to redhat registry using docker. Use the following command to login to registry.redhat.com. 
+[source,bash]
+----
+$ docker login registry.redhat.io
+----
+Refer link:https://access.redhat.com/RegistryAuthentication[Registry Authentication] for more information about registry login.
```
Comment (ianf77) [inline stories/topics/proc-aws-upgrade-pull-container-image.adoc:15]: As above.

File: stories/topics/proc-gcp-backup-container-image.adoc [hunk 10]
```diff
@@ -5,6 +5,16 @@
...
 * Pull the docker image for the `ansible-on-clouds-ops` 2.3 container with the same tag as the foundation deployment.
 +
+[NOTE]
+====
+Before Pulling the docker image, make sure you are logged in to redhat registry using docker. Use the following command to login to registry.redhat.com. 
+[source,bash]
+----
+$ docker login registry.redhat.io
+----
+Refer link:https://access.redhat.com/RegistryAuthentication[Registry Authentication] for more information about registry login.
```
Comment (ianf77) [inline stories/topics/proc-gcp-backup-container-image.adoc:15]: As above.

File: stories/topics/proc-gcp-from-stack-pull-container-image.adoc [hunk 11]
```diff
@@ -5,6 +5,17 @@
...
 
++
+[NOTE]
+====
+Before Pulling the docker image, make sure you are logged in to redhat registry using docker. Use the following command to login to registry.redhat.com.
+[source,bash]
+----
+$ docker login registry.redhat.io
+----
+Refer link:https://access.redhat.com/RegistryAuthentication[Registry Authentication] for more information about registry login.
```
Comment (ianf77) [inline stories/topics/proc-gcp-from-stack-pull-container-image.adoc:16]: As above.

File: stories/topics/proc-gcp-pull-backup-container-image.adoc [hunk 12]
```diff
@@ -5,6 +5,17 @@
...
 
++
+[NOTE]
+====
+Before Pulling the docker image, make sure you are logged in to redhat registry using docker. Use the following command to login to registry.redhat.com.
+[source,bash]
+----
+$ docker login registry.redhat.io
+----
+Refer link:https://access.redhat.com/RegistryAuthentication[Registry Authentication] for more information about registry login.
```
Comment (ianf77) [inline stories/topics/proc-gcp-pull-backup-container-image.adoc:16]: As above.

File: stories/topics/proc-gcp-upgrade-pull-container-image.adoc [hunk 13]
```diff
@@ -6,6 +6,16 @@
 * Pull the docker image for the `ansible-on-clouds-ops` 2.3 container with the same tag as the version you want to upgrade to.
 +
 [NOTE]
+====
+Before Pulling the docker image, make sure you are logged in to redhat registry using docker. Use the following command to login to registry.redhat.com.
+[source,bash]
+----
+$ docker login registry.redhat.io
+----
+Refer link:https://access.redhat.com/RegistryAuthentication[Registry Authentication] for more information about registry login.
```
Comment (ianf77) [inline stories/topics/proc-gcp-upgrade-pull-container-image.adoc:15]: As above.

**Review by ianf77 — APPROVED** [review]
LGTM

**Comment by chrisahl** [comment]
@ianf77 I assume you are the one who does the merging for these doc PRs?

**Comment by ianf77** [comment]
Either myself or Aine.

**Pull request merged**